pysd/py_backend/vensim/vensim2py.py

```diff
@@ -361,13 +360,16 @@ def visit_data_definition(self, n, vc):
             self.kind = "data"
 
         def visit_test_definition(self, n, vc):
+            # TODO: add test for test
```

Comment:
Are these todo's still open? (and ln 371/2, 399?)

Reply:
Yes, this are two features that I didn't add and we have no test that cover this lines.
I tried to add tests for those lines but I don't know how the original files should look like.
Line 371 is quite more complicated as it is related about how the subscripts are defined.